stable/filebeat/values.yaml

```diff
@@ -75,6 +77,12 @@ resources: {}
   #  cpu: 100m
   #  memory: 100Mi
 
+nodeSelector: {}
+
+tolerations: []
```

Comment:
please put the "opinionated" toleration here as users will expect this to collect all logs.

Reply:
@sstarcher fixed

Reply:
What about the other tolerations?
```
-      - key: node-role.kubernetes.io/master
-        operator: Exists
-        effect: NoSchedule
```
You have missed all the elements of the original toleration

Reply:
I don't think you are correct.
The - indicates it's an array and the whole hash i pasted is one element of that array.
You have only included one part of that sub hash.